### PR TITLE
Adding DOB standardization functions and tests to Standardization container

### PR DESCRIPTION
**Pull request by robertmitchellv**
# PULL REQUEST

## Summary
Adding the date standardization functions to the Standardization container. Given that we're moving away from the SDK and adding functions to the `utils.py` script within the application files there may need to be some changes to ensure the atomic versions of SDK functions and their FHIR counterparts can be leveraged in a modular way. This PR proposes a wrapper to apply a function to a bundle using a FHIRPath.

## Related Issue
Fixes https://github.com/CDCgov/phdi-azure/issues/369

## Checklist

- [x] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/standardization/tests/test_standardization.py [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+    read_json_from_assets,
+    standardize_name,
+    _standardize_date,
+    _validate_date,
+)
 
 client = TestClient(app)
 
 
+@pytest.fixture
```
Comment (emmastephenson) [inline containers/standardization/tests/test_standardization.py:15]: ooh nice! I like the fixture

File: containers/standardization/tests/test_standardization.py [hunk 2]
```diff
@@ -36,3 +47,51 @@ def test_standardize_name():
         "PAUL BUNYAN",
         "JRRTOLKIE87N 999",
     ]
+
+
+@pytest.mark.parametrize(
```
Comment (emmastephenson) [inline containers/standardization/tests/test_standardization.py:52]: 🤩 wicked cool, love the parameterized tests!

**Review by emmastephenson — APPROVED** [review]
LGTM, thanks for the through testing!

File: containers/standardization/app/utils.py [hunk 3]
```diff
@@ -1,12 +1,75 @@
...
+
+FHIR_DATE_FORMAT = "%Y-%m-%d"
+FHIR_DATE_DELIM = "-"
 
 
 def read_json_from_assets(filename: str):
     return json.load(open((pathlib.Path(__file__).parent.parent / "assets" / filename)))
 
 
+def is_fhir_bundle(bundle) -> bool:
```
[the remark this answers is not in the document]
Comment (robertmitchellv) [reply containers/standardization/app/utils.py:19]: i really like this idea a lot! would love to work on that 🚀

File: containers/standardization/app/utils.py [hunk 4]
```diff
@@ -1,12 +1,75 @@
...
+
+from detect_delimiter import detect
+from fhirpathpy import evaluate as fhirpath_evaluate
+
+
+FHIR_DATE_FORMAT = "%Y-%m-%d"
+FHIR_DATE_DELIM = "-"
 
 
 def read_json_from_assets(filename: str):
```
[the remark this answers is not in the document]
Comment (robertmitchellv) [reply containers/standardization/app/utils.py:15]: yeah, i think this service will hopefully provide the impetus for deciding what goes in a new SDK and what stays at the container level

**Pull request merged**